packages/mobile/src/account/Settings.tsx

```diff
@@ -414,15 +423,85 @@ export class Account extends React.Component<Props, State> {
     }
   }
 
-  render() {
+  getLinkBankAccountSettingItem() {
     const {
-      t,
-      i18n,
-      numberVerified,
-      verificationPossible,
+      kycStatus,
       linkBankAccountEnabled,
+      hasLinkedBankAccount,
       mtwAddress,
+      linkBankAccountStepTwoEnabled,
+      t,
     } = this.props
+
+    // Not enabled
```

Comment:
The meat of this PR is in all of these if statements. I could have made it in less lines of code, but I felt it was better to be more verbose and explicit about the business logic

Reply:
i think being explicit always > less lines of code